.github/workflows/jerigon-zero.yml

```diff
@@ -10,7 +10,6 @@ on:
     branches:
       - "**"
 
-
```

Comment:
Yep agreed. All these changes occurred on file save. I don't imagine another IDE would undo these diffs - e.g. removing unnecessary whitespace and newlines. In which case I would suggest we leave these diffs. Otherwise I can try turn off auto format for yaml files

Reply:
https://github.com/0xPolygonZero/zk_evm/issues/743